tests/test_timer.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import json
+import unittest
+
+import azure.functions.timer as timer
+from azure.functions.meta import Datum
+
+
+class TestTimer(unittest.TestCase):
+    def test_TimerRequestConverter_decode(self):
```

Comment:
We might need to add some more tests. Check the test cases in other triggers. 
e.g. test_timer_input_type
test_timer_output_type
test_timer_decode
test_timer_initialize_without_args
test_timer_initialize_all_args

Reply:
I have added few more tests. thanks